microsoft-365/admin/manage/upgrade-distribution-lists.md

```diff
@@ -98,8 +98,8 @@ Upgrade-DistributionGroup -DlIdentities dl1@contoso.com
 You can also pass multiple DLs as a batch and upgrade them together:
 
 ```PowerShell
-Upgrade-DistributionGroup -DlIdentities \<DL SMTP address1\>, \< DL SMTP address2\>,
-\< DL SMTP address3\>, \< DL SMTP address 4\>
+Upgrade-DistributionGroup -DlIdentities <DL SMTP address1>, < DL SMTP address2>,
+< DL SMTP address3>, < DL SMTP address 4>
```

Comment:
The spacing does not seem right. Should there be a space after the first angle in both instances? And also, "address3" but "address 4". Should there be a space between "address" and the numeral?

Reply:
I'll fix that

Reply:
It has been updated now.